### PR TITLE
plugin.video.contv 4.0.1

### PR DESCRIPTION
**Pull request by learningit**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by learningit** [comment]
Requires matrix-script.module.t1mlib-4.02 to be merged so Travis doesn't complain

File: plugin.video.contv/resources/lib/scraper.py [hunk 1]
```diff
@@ -39,10 +39,12 @@ def getAddonShows(self,url,ilist):
           infoList['Plot'] = b.get("description")
           if b["type"] == 'vod':
               infoList['mediatype'] = 'movie'
-              ilist = self.addMenuItem(name,'GV', ilist, url, thumb, fanart, infoList, isFolder=False)
+              contextMenu = [('Add To Library',''.join(['RunPlugin(',sys.argv[0],'?mode=AM&url=',url,')']))]
```
Comment (enen92) [inline plugin.video.contv/resources/lib/scraper.py]: localized string please

File: plugin.video.contv/resources/lib/scraper.py [hunk 2]
```diff
@@ -39,10 +39,12 @@ def getAddonShows(self,url,ilist):
...
+              ilist = self.addMenuItem(name,'GV', ilist, url, thumb, fanart, infoList, isFolder=False, cm=contextMenu)
           elif b["type"] == 'episodic':
               infoList['mediatype'] = 'tvshow'
-              ilist = self.addMenuItem(name,'GE', ilist, url, thumb, fanart, infoList, isFolder=True)
+              contextMenu = [('Add To Library',''.join(['RunPlugin(',sys.argv[0],'?mode=AS&url=',url,')']))]
```
Comment (enen92) [inline plugin.video.contv/resources/lib/scraper.py]: localized string whenever you have strings that are shown in the UI to the user

**Review by enen92 — CHANGES_REQUESTED** [review]
missing localization

**Review by enen92 — APPROVED** [review]

**Pull request merged**